lib/netplay/netplay.cpp

```diff
@@ -219,6 +214,25 @@ void NETGameLocked(bool flag)
 	debug(LOG_NET, "Passworded game is %s", NetPlay.GamePassworded ? "TRUE" : "FALSE");
 }
 
+void NETsetLobbyOptField(const char *Value, const NET_LOBBY_OPT_FIELD Field)
+{
+	switch (Field)
+	{
+		case NET_LOBBY_OPT_FIELD::GNAME:
+			sstrcpy(gamestruct.name, Value);
+			break;
+		case NET_LOBBY_OPT_FIELD::MAPNAME:
+			sstrcpy(gamestruct.mapname, Value);
```

Comment:
Try `sstrcpy(gamestruct.mapname, formatGameName(Value).toUtf8().c_str());`

Then in multiint.cpp find function `formatGameName()`, remove the `static`, and add a declaration in multiint.h `WzString formatGameName(WzString name);`

This will fix the tech levels always appearing as T1.